source/examples/rapids-sagemaker-hpo/notebook.ipynb

```diff
@@ -805,10 +805,13 @@
     "\n",
     "# install a few more dependencies\n",
     "RUN conda install --yes -n base \\\n",
+    "    {{ rapids_conda_channels }} \\\n",
     "        cupy \\\n",
+    "        dask-ml \\\n",
     "        flask \\\n",
     "        protobuf \\\n",
-    "        sagemaker\n",
+    "        rapids-dask-dependency=${{ rapids_version }} \\\n",
```

Comment:
To ensure `dask-ml` installation doesn't result in upgrading / downgrading `dask` and `distributed` and therefore changing the installed version of RAPIDS libraries.